geomstats/geometry/hyperbolic.py

```diff
@@ -272,11 +271,10 @@ def _intrinsic_to_extrinsic_coordinates(point_intrinsic):
         Parameters
         ----------
         point_intrinsic : array-like, shape=[n_samples, dimension]
-            Point in hyperbolic space in intrinsic coordinates.
 
         Returns
         -------
-        point_extrinsic : array-like, shape=[n_samples, dimensio.n + 1]
+        point_extrinsic : array-like, shape=[n_samples, dimension + 1]
             Point in hyperbolic space in extrinsic coordinates
```

Comment:
Missing "." at the end of this line (I know it's not your fault, but let's put it since we're here.)